src/app/workflow/view/view.service.ts

```diff
@@ -153,9 +156,12 @@ export class ViewService {
       confirmationButtonText: 'Snapshot Version',
       cancelButtonText: 'Cancel'
     };
-    this.confirmationDialogService.openDialog(dialogData, bootstrap4mediumModalSize).subscribe(confirmationResult => {
+    this.confirmationDialogService.openDialog(dialogData, bootstrap4mediumModalSize).subscribe((confirmationResult: Boolean) => {
       if (confirmationResult) {
-        this.updateWorkflowToSnapshot(workflow, version, () => this.alertService.detailedSuccess('Snapshot successfully created!'));
+        this.updateWorkflowToSnapshot(workflow, version, () => {
+          this.alertService.detailedSuccess(`A Snapshot was successfully requested for workflow
```

Comment:
This is nit-picky, but:

1. I would not have a capital S in snapshot
2. I would say created or issued instead of requested -- "successfully requested" means, to me, that the request was accepted, but not necessarily that it was fulfilled.
3. I'd probably remove "successfully".